packages/neuron-ui/src/components/PricePanel/index.tsx

```diff
@@ -0,0 +1,170 @@
+import { localNumberFormatter, shannonToCKBFormatter } from 'utils'
+import RingProgressBar from 'widgets/RingProgressBar'
+import TextField from 'widgets/TextField'
+import { Transfer } from 'widgets/Icons/icon'
+import DropdownWithCustomRender from 'widgets/DropdownWithCustomRender'
+import { useTranslation } from 'react-i18next'
+import { appState, useState as useGlobalState } from 'states'
+
+import { useGetBatchGeneratedTx } from 'components/Send/hooks'
+import { DEFAULT_COUNT_DOWN } from 'utils/const'
+
+import styles from './pricePanel.module.scss'
+
+interface PricePanelProps {
+  field: string
+  price: string
+  onPriceChange: React.EventHandler<React.SyntheticEvent>
+}
+enum PriceTypeEnum {
+  Custom = 'custom',
+  Standard = 'standard',
+}
+const DEFAULT_PRICE_ARRAY = ['1000', '2000', '3000']
+
+const PricePanel: React.FunctionComponent<PricePanelProps> = ({ price, field, onPriceChange }: PricePanelProps) => {
+  const [t] = useTranslation()
+  const [type, setType] = useState<PriceTypeEnum>(PriceTypeEnum.Standard)
+  const [dropdownValue, setDropdowValue] = useState(DEFAULT_PRICE_ARRAY[1])
+  const [feeRateValueArray, setFeeRateValueArray] = useState([])
+  const [priceArray, setPriceArray] = useState(DEFAULT_PRICE_ARRAY)
+  const [currentModeIdx, setCurrentModeIdx] = useState<number>(1)
+  const [inputError, setInputError] = useState<string | null>(null)
+  const {
+    app: {
+      send = appState.send,
+      feeRateStatics: { suggestFeeRate = 0 },
+      countDown,
+    },
+    wallet: { id: walletID = '' },
+  } = useGlobalState()
+
+  const isStandard = type === PriceTypeEnum.Standard
+  const label = isStandard ? t('price-switch.price') : t('price-switch.customPrice')
+  const percent = (countDown / DEFAULT_COUNT_DOWN) * 100
+
+  const handleDropdownChange = useCallback(
+    (e: { value: string; index?: number }) => {
+      const { value: dropdownChangedValue, index = 1 } = e
+      setDropdowValue(dropdownChangedValue)
+      setCurrentModeIdx(index)
+
+      onPriceChange?.({ target: { value: dropdownChangedValue } } as any)
+    },
+    [setDropdowValue, onPriceChange]
+  )
+
+  const handleInputChange = useCallback(
+    (e: React.SyntheticEvent<HTMLInputElement>) => {
+      const { value: inputChangedValue } = e.currentTarget
+      const handledValue = inputChangedValue.replace(/,/g, '')
+
+      setInputError(Number(handledValue) < 1000 ? t('price-switch.errorTip', { minPrice: 1000 }) : null)
+      onPriceChange?.(e)
+    },
+    [onPriceChange]
+  )
+
+  const feeValuesArray = useMemo(
+    () =>
+      feeRateValueArray?.map((item: any) => ({
+        feeRateValue: item.feeRateValue,
+        value: shannonToCKBFormatter(item.feeValue),
+      })),
+    [feeRateValueArray]
+  )
+
+  useEffect(() => {
+    if (isStandard && !priceArray.includes(dropdownValue)) {
+      handleDropdownChange({ value: priceArray[currentModeIdx], index: currentModeIdx })
+    }
+  }, [priceArray])
```

Comment:
`isStandard`, `handleDropdownChange` and `currentModeIdx` are missing in the deps